README.md

```diff
@@ -57,4 +57,27 @@ If you are on call or asked to fix an immediate issue, check out our doc on
 [how support works](/playbooks/support.md#readme) and reference our
 [support wiki](https://github.com/artsy/potential/wiki) 🔒 for up-to-date playbooks on how to solve issues.
 
-<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
+
+## About Artsy
+
+<a href="https://www.artsy.net/">
+  <img align="left" src="https://avatars2.githubusercontent.com/u/546231?s=200&v=4"/>
+</a>
+
+This project is supported by [Artsy][footer_website], a marketplace for buying
+and selling art online. Because one of our most beloved [Engineering
+Principles][footer_principles] is being [Open Source by Default][footer_open],
+we share a lot of how we work with the open source community. You can learn more
+about our open source projects from [our blog][footer_blog] and following
```

Comment:
I think we're missing a `by` between the blog and twitter, and a comma would make it more parseable. Suggested changes in bold:
> You can learn more about our open source projects from our blog and **by** following `@ArtsyOpenSource` **,** or check out our API

I also feel like "check out our API" is an incomplete thought but I'm not sure how to round it out. "check out our API in order to.../if you're interested in.../to explore 1M+ artworks..."?